docs/releases/1.18-NOTES.md

```diff
@@ -42,7 +42,9 @@
 
 * Support for the Debian 8 (Jessie) OS distribution has been removed.
 
-* The Docker `health-check` service is now disabled by default. It shouldn't be needed anymore, but it can still be enabled by setting `spec.docker.healthCheck: true`. It is recommended to also check [node-problem-detector](https://github.com/kubernetes/node-problem-detector) and [draino](https://github.com/planetlabs/draino) as replacements. See Required Actions below.
+* The Docker `health-check` service has been disabled by default. It shouldn't be needed anymore, but it can still be enabled by setting `spec.docker.healthCheck: true`. It is recommended to also check [node-problem-detector](https://github.com/kubernetes/node-problem-detector) and [draino](https://github.com/planetlabs/draino) as replacements. See Required Actions below.
+
+* Network and internet access for `docker run` containers has been disabled by default. To re-enable, set `spec.docker.ipTables: true` and `spec.docker.ipMasq: true`.
```

Comment:
Can we reword this first sentence to make it not sound like we decided to just make this change on its own? maybe briefly mention what this change is a side-effect of.

> As a result of ___, containers launched directly via `docker run` will no longer have network and internet access. To re-enable, ...

Reply:
I think it sounds better now.